gem2rpm/foreman_plugin.spec.erb

```diff
@@ -77,6 +76,13 @@ BuildRequires: gcc
 BuildArch: noarch
 <% end -%>
 Provides: foreman-plugin-%{plugin_name} = %{version}
+<% if has_assets || has_webpack -%>
+<% for d in spec.runtime_dependencies -%>
+<% for req in d.requirement -%>
+BuildRequires: rubygem(<%= d.name %>)<%= " #{req}" unless req.empty? %>
+<% end -%>
+<% end -%>
+<% end -%>
```

Comment:
Sad :/ What step of the build related to assets is it that ends up requiring all the runtime dependencies present? (my memory fails me)

Reply:
`foreman-rake assets:precompile[$plugin]`